src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -807,11 +808,33 @@ public String build(final Path directory, final String name, final ProgressHandl
      resource = resource.queryParam("t", name);
     }
 
+    log.debug("Auth Config {}", authConfig);
+
+    // Convert auth to X-Registry-Config format
+    AuthRegistryConfig registryConfig = null;
+    if (authConfig == null) {
+      registryConfig = new AuthRegistryConfig("", "", "", "", "", "");
```

Comment:
Would be better to have an `AuthRegistryConfig.EMPTY` static variable rather than constructing a new empty instance each time.